packages/components/src/local/organisms/setting-channels/index.tsx

```diff
@@ -152,14 +147,7 @@ export const SettingChannels: React.FC<PropTypes> = ({
       setLocalChannelUpdates(localChannelUpdates)
     }
 
-    const handleChangeRow = (nextItems: Array<RowItem>, /* _itKey: number, */ participant: CoreParticipant): Array<RowItem> => {
-      // TODO: this next block was failing, because the [2] item was empty. Find out what the code
```

Comment:
Hello @lebaphi  - do you recognise what this commented out block is doing?  It was failing at runtime. I couldn't see what it was doing, so commented it out.  That may have been a silly thing to do 😉